psalm-baseline.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<files psalm-version="4.3.2@57b53ff26237074fdf5cbcb034f7da5172be4524">
+<files psalm-version="4.1.1@16bfbd9224698bd738c665f33039fade2a1a3977">
```

Comment:
please use the same version as the dependancy here

Reply:
My bad, this targets 2.8 and Psalm is indeed in 4.1.1 there. The baseline was generated with a more up to date version before.

Reply:
Woopsie